src/Path/Container/AddNewProducts.jsx

```diff
@@ -10,1202 +10,1259 @@ import { getCities } from "../../Services/citiesApi";
+      setImage((prevImages) => [...prevImages, ...allowedFiles]);
+    } else {
+      setLocalImages((prevImages) => [...prevImages, ...validImages]);
+      setImage((prevImages) => [...prevImages, ...validImages]);
+    }
 
-        // Filter for only valid images
-        const validImages = newFiles.filter(file => file.type.startsWith("image/"));
-        const invalidFiles = newFiles.filter(file => !file.type.startsWith("image/"));
+    setLocalImageOrPdf(true);
+    e.target.value = null; // Reset input
+  }
 
-        // If there are invalid files (e.g., not images), show an alert
-        if (invalidFiles.length > 0) {
-            alert(t("imagePdfAlert"));
-            return;
-        }
+  const [localImages, setLocalImages] = useState([]);
+  const handleMultipleInputChange = (event) => {
+    const newFiles = Array.from(event.target.files);
 
-        const currentImageCount = image ? image.length : 0;
-        const totalImages = currentImageCount + validImages.length;
+    // Filter for only valid images
+    const validImages = newFiles.filter((file) =>
+      file.type.startsWith("image/")
+    );
+    const invalidFiles = newFiles.filter(
+      (file) => !file.type.startsWith("image/")
+    );
 
-        if (totalImages > 3) {
-            alert(t("productImageNumberAlert"));
-            const allowedFiles = validImages.slice(0, 3 - currentImageCount);
-            setLocalImages((prevImages) => [...prevImages, ...allowedFiles]);
-            setImage((prevImages) => [...prevImages, ...allowedFiles]);
-        } else {
-            setLocalImages((prevImages) => [...prevImages, ...validImages]);
-            setImage((prevImages) => [...prevImages, ...validImages]);
-        }
+    // If there are invalid files (e.g., not images), show an alert
+    if (invalidFiles.length > 0) {
+      alert(t("imagePdfAlert"));
+      return;
+    }
 
-        event.target.value = null; // Reset input
-    };
+    const currentImageCount = image ? image.length : 0;
+    const totalImages = currentImageCount + validImages.length;
+
+    if (totalImages > 3) {
+      alert(t("productImageNumberAlert"));
+      const allowedFiles = validImages.slice(0, 3 - currentImageCount);
+      setLocalImages((prevImages) => [...prevImages, ...allowedFiles]);
+      setImage((prevImages) => [...prevImages, ...allowedFiles]);
+    } else {
+      setLocalImages((prevImages) => [...prevImages, ...validImages]);
+      setImage((prevImages) => [...prevImages, ...validImages]);
+    }
 
-    const handleUpdateMultipleInputChange = (e) => {
-        const newFiles = Array.from(e.target.files);
+    event.target.value = null; // Reset input
+  };
 
-        const validImages = newFiles.filter(file => file.type.startsWith("image/"));
-        const invalidFiles = newFiles.filter(file => !file.type.startsWith("image/"));
+  const handleUpdateMultipleInputChange = (e) => {
+    const newFiles = Array.from(e.target.files);
 
```

Comment:
Is these two functions -> handleMultipleInputChange() and handleUpdateMultipleInputChange() doing the same functionality?? if yes, why two?

Reply:
will remove one function